dom/src/MainDOMSource.ts

```diff
@@ -215,3 +265,11 @@ export class MainDOMSource implements DOMSource {
   public isolateSource: (source: DOMSource, scope: string) => DOMSource = isolateSource;
   public isolateSink: (sink: Stream<VNode>, scope: string) => Stream<VNode> = isolateSink;
 }
+
+export interface MainDOMSourceOptions extends SourceOptions {
```

Comment:
Could we move this up?
The order of stuff in any file should be "anything that is used in line `x` should be defined in some line `x - n`".
Imports at the top, exports at the bottom.